sync-core/src/test/java/com/cloudant/sync/replication/BasicReplicatorMockTest.java

```diff
@@ -256,8 +258,17 @@ public void start_stoppedThenRestarted() throws Exception {
         verify(mockListener).complete(rc);
         verify(mockListener, never()).error(any(ReplicationErrored.class));
 
+        doAnswer(new Answer() {
```

Comment:
I'm not really sure, I didn't want to mess with the other tests since I'm not sure how the set up their replicator to produce errors or if the a mocked class methods just return.